app/services/users/profile.rb

```diff
@@ -96,7 +96,7 @@ def claims
           military_history: Vet360Policy.new(user).military_access?,
           payment_history: BGSPolicy.new(user).access?(log_stats: false),
           personal_information: MPIPolicy.new(user).queryable?,
-          rating_info: EVSSPolicy.new(user).access?
+          rating_info: LighthousePolicy.new(user).rating_info_access?
```

Comment:
These policies don't look equivalent.

```ruby
# lighthouse_policy.rb

  def access?
    user.icn.present? && user.participant_id.present?
  end
```

```ruby
# evss_policy.rb

  def access?
    if user.edipi.present? && user.ssn.present? && user.participant_id.present?
      log_success('access')
    else
      log_failure('access')
    end
  end
```

I am relatively new-ish to the team, so there's probably some info I'm missing that you could enlighten me with. As I understand it, the change you're introducing here deprecates checking for the presence of a user's `edipi` and `ssn` attributes in favor of checking the `icn` (both policies check `participant_id`). Can you provide some context around that change to help me understand? Thanks.

Reply:
Hi @dickdavis.  This policy change is the result of the migration from EVSS to Lighthouse.  The EVSS authorization required the `edipi`, `ssn`, and `participant_id`, but Lighthouse has confirmed that in order to make the request on behalf of an authenticated user, only the ICN is required.